doc/sphinx/addendum/micromega.rst

```diff
@@ -194,10 +194,11 @@ a proof.
 .. tacn:: nra
    :name: nra
 
-   This tactic is an *experimental* proof procedure for non-linear
-   arithmetic. The tactic performs a limited amount of non-linear
-   reasoning before running the linear prover of :tacn:`lra`. This pre-processing
-   does the following:
+   The status of this tactic is experimental.
```

Comment:
You forgot `.. warning::` there. That said, I would not mark this tactic as experimental. It is not like its behavior might suddenly change in the next version.